integration-tests/docker-compose.anvil.yml

```diff
@@ -0,0 +1,39 @@
+services:
+  anvil:
+    image: ghcr.io/foundry-rs/foundry:latest
+    platform: linux/amd64
+    command:
+      - anvil --host 0.0.0.0
+    ports:
+      - 8545:8545
+
+  deploy-sntv2:
+    platform: linux/amd64
+    environment:
+      - API_KEY_ETHERSCAN=""
+    depends_on:
+      - anvil
+    build:
+      context: .
+      dockerfile: Dockerfile
+      args:
+        github_org: status-im
+        github_repo: status-network-token-v2
+    entrypoint: ["forge", "script", "script/Deploy.s.sol", "--broadcast", "--fork-url", "http://anvil:8545", "--private-key=0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80"]
+
+  deploy-communities-contracts:
+    platform: linux/amd64
+    environment:
+      - API_KEY_ETHERSCAN=""
+      - API_KEY_ARBISCAN=""
+      - API_KEY_OPTIMISTIC_ETHERSCAN=""
```

Comment:
Why are we setting this to nothing? Wouldn't it make more sense just to skip this whole section? Or does setting them to empty strings does something specific? And if it does, it probably should be commented upon.

Reply:
it does, having env var is required (crashes without) but values aren't used. will add comments